docroot/themes/contrib/civictheme/includes/node.inc

```diff
@@ -114,6 +114,10 @@ function _civictheme_preprocess_page__node(&$variables) {
 function _civictheme_preprocess_node__civictheme_slider_slide(&$variables) {
   /** @var \Drupal\node\NodeInterface $node */
   $node = $variables['node'];
+  // @todo add theme field to reference slide.
+  $paragraph = $node->_referringItem->getEntity();
+  $variables['theme'] = $paragraph->civic_theme;
```

Comment:
Small fix until we add theme field to reference cards

Reply:
please do not use `civic_` prefix with any of the code. Only `civictheme_` is allowed. thank you

Reply:
Thanks for reminder